src/cancellation.js

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
Maybe just exclude them in `tsconfig`? It seems all `js` from src n oneed to be check, am I right?

Reply:
We should check all the JS in order to detect and fix discrepancies between the current manual JS-SDK implementation and the current  openAPI spec. Sometimes the error will be in one side and sometimes in the other side. That's why I decided to start with an explicit `@ts-nocheck` in the source files. It is a kind of TODO mark that reminds us to fix our discrepancies. 

Furthermore, there is an RFC to auto-generate  the JS-SDK code in the next cycles. That would implicitly force us  to remove all those temporary markers.